src/frontend/test_runner/tests/testdata/tpch.yaml

```diff
@@ -1863,19 +1863,18 @@
         BatchHashAgg { group_keys: [$0], aggs: [count, sum($1)] }
           BatchExchange { order: [], dist: HashShard([0]) }
             BatchProject { exprs: [Substr($0, 1:Int32, 2:Int32), $1] }
-              BatchExchange { order: [], dist: AnyShard }
```

Comment:
I think there is a bug. if our batch execution can handle exchange with `AnyShard`? @liurenjie1024

Reply:
Currently no.